apps/core/scenes/search/Search.js

```diff
@@ -58,20 +65,24 @@ class Search extends React.Component<Props> {
       returnDateTo,
       tripType,
     } = this.props;
-    this.props.navigation.navigate(Routes.RESULTS, {
-      travelFrom: travelFrom?.locationId,
-      travelTo: travelTo?.locationId,
-      travelFromName: travelFrom?.name,
-      travelToName: travelTo?.name,
-      dateFrom: format(dateFrom, DATE_FORMAT),
-      dateTo: format(dateTo, DATE_FORMAT),
-      ...(tripType === 'return'
-        ? {
-            returnDateFrom: format(returnDateFrom, DATE_FORMAT),
-            returnDateTo: format(returnDateTo, DATE_FORMAT),
-          }
-        : {}),
-    });
+    if (travelFrom == null && travelTo == null) {
```

Comment:
What if there is no `travelFrom `?

Reply:
Good point. On Kiwi FE, the button is actually disabled in that case. So I don't know if alerting was the best UX solution.